README.adoc

```diff
@@ -342,7 +407,20 @@ Verifying a pact between pact between Inventory (1.0-SNAPSHOT) and System
 [INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.835 s - in it.io.openliberty.guides.system.SystemBrokerIT
 ...
 ```
-After the tests are complete, refresh the Pact Broker webpage at the http://localhost:9292/[^] URL to confirm that there's now a timestamp in the last verified column:
+
+// Static guide instruction
+ifndef::cloud-hosted[]
+After the tests are complete, refresh the Pact Broker webpage at the http://localhost:9292/[^] URL to
+confirm that there's now a timestamp in the last verified column:
```

Comment:
confirm that there's now a timestamp in the last verified column:
-->
confirm that the last verified column now shows a timestamp: